Automattic/wp-calypso
# Change tooltip in onboarding for virtual themes

Related to https://github.com/Automattic/dotcom-forge/issues/1654 but see also paYJgx-30q-p2#comment-3246 for further context

## Proposed Changes

Premium themes and virtual themes made of style variations of premium and non-premium themes require the Premium or above plan to use. These themes and v-themes display a 'Premium' badge that has a tooltip explaining what premium means.

For virtual themes made of style variations on sites that are on a lower plan than premium this tooltip will now be more relevant.


## Testing Instructions




1. Use calypso.live link from the comment below
2. Go to /setup/site-setup/designSetup?siteSlug=YOURFREESITE.wordpress.com&flags=virtual-themes/onboarding
3. Hover over the Premium badge for a virtual style variation theme (it'll end in " - VARIATION_NAME) and check that the tool tip makes sense
4. Do the same for a regular premium theme, the tooltip should be as production
5. Reload the page using a premium site
6. See how everything is as in production - the virtual style variation themes don't have a premium badge, the premium themes have the same label as production.

## Pre-merge Checklist
